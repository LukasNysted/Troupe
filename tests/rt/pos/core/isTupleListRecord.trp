let val t = (1,2,3)
    val l = [1,2,3]
    val r = {a=1, b=2, c=3}
in (isTuple t, isList l, isRecord r,
    isList t, isRecord l, isTuple r,
    isRecord t, isTuple l, isList r,
    isTuple 0, isList 0, isRecord 0,
    isTuple "", isList "", isRecord "")
end