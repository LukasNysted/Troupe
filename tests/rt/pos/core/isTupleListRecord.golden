2023-07-06T14:40:42.642Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: (true@{}%{}, true@{}%{}, true@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{}, false@{}%{})@{}%{}
